envisage/developer/code_browser/code_browser.py

```diff
@@ -2,15 +2,20 @@
 
 
 # Standard library imports.
-import cPickle, imp, logging, os, stat, warnings
+import sys, imp, logging, os, stat, warnings
+if sys.version_info[0] >= 3:
```

Comment:
This, along with similar checks are duplicated throughout the PR. Please create an `envisage._compat` module collecting this logic and import the required objects from there.